content/cloud-docs/api-docs/teams.mdx

```diff
@@ -38,6 +38,8 @@ page_title: Teams - API Docs - Terraform Cloud and Terraform Enterprise
 
 -> **Note:** Team management is a paid feature, available as part of the **Team** upgrade package. [Learn more about Terraform Cloud pricing here](https://www.hashicorp.com/products/terraform/pricing).
```

Comment:
What if we combined these two notes. Having two notes in a row like this can be visually overwhelming for users. Combining them into one note, especially since they are related (both paid features) I think is an improvement. Thoughts?

```suggestion
-> **Note:** Team management is a paid feature, available as part of the **Team** upgrade package. Single sign-on is also a paid feature, available as part of the **Business** upgrade package. Refer to [Terraform Cloud pricing](https://www.hashicorp.com/products/terraform/pricing) for details.
```